README.md

```diff
@@ -98,6 +98,10 @@ Then refresh the web page you want to profile and refresh the control panel to s
 
 Then click on the report in the list and enjoy the [analysis screen](#analysis-screen).
 
+Profiling can also be triggered with Curl as shown in this example:
```

Comment:
@8ctopus IMO it should be placed right after
> Then switch on "Enabled". At this point profiling is enabled for the current domain and your current browser session through a set of dedicated cookies.

Reply:
DONE